### ISSUE TITLE
Support optional macaddress poperty in vlan definition

### ISSUE THREAD

**Issue by DasUrmel** [openmediavault/openmediavault#1306]
**Is your feature request related to a problem? Please describe.**
dhcp in vlans based on a bond do not work because OMV does not support to enter an (optional) macaddress in the vlan definition. As a result all vlans in the bond share the same mac address.
In dhcp server (like pihole) you are not allowed to define multiple IP for the same mac address, even when in different networks.

**Describe the solution you'd like**
I'd like the definition of vlans enhanced by an optional property "macaddress".
See my thread in the forum with detailed information:
https://forum.openmediavault.org/index.php?thread/43201-vlan-and-mac-address/

**Describe alternatives you've considered**
It works when defining a property macaddress manually in the netplan yaml files. But they are deleted when reconfigured by OMV.

**Additional context**
None

**Comment by ryecoaaron** [comment]
Not to sidetrack your issue but OMV only deletes the files with openmediavault in the name in /etc/netplan/ - https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault/srv/salt/omv/deploy/systemd-networkd/10cleanup.sls#L20.  So, you could define this manually.

**Comment by DasUrmel** [comment]
You're right. Configuring everything "under the hood" works just fine after renaming the files and deleting vlans and bond in OMV.
To my opinion OMV has a different approach in assisting the admin and not requiring him deep knowledge in linux networking.

**Comment by ryecoaaron** [comment]
I agree but OMV cannot do everything. 99.5% of OMV users would not use this. That is why Volker asks for code contributions due to limited time and ability to test.
